Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:latest
+RUN npm install -g http-server
+RUN mkdir /public
+WORKDIR /public
+
+EXPOSE 8080
+CMD ["http-server"]
```

Comment:
On my machine, this takes 67 seconds to build and results in a 1GB image :( Are there better alternatives?

One option is `python:3.10-alpine`, then use  `python -m http.server`. In fact, that python command will work on any computer with python3 installed (which should include all macos computers out-of-the-box), no containers needed.